notes.org

```diff
@@ -37,24 +37,29 @@
 		 let posts_in_blog = query!(Post, &conn, blog = {blog}).load(&conn)
 	 }
 	 #+END_SRC
+
 	 struct Posts
+
+	 #+BEGIN_SRC sql
 	 CREATE TABLE posts (
-  id SERIAL PRIMARY KEY,
-  title VARCHAR NOT NULL,
-  body TEXT NOT NULL,
-  published BOOLEAN NOT NULL DEFAULT 'f'
-)
+		id SERIAL PRIMARY KEY,
+		title VARCHAR NOT NULL,
+		body TEXT NOT NULL,
+		published BOOLEAN NOT NULL DEFAULT 'f'
+	 )
+	 #+END_SRC
+
 * Roadmap
 ** vNext
-	 + foreign key constraints in db, cascade setting
+	 + foreign key constraints in db, cascade setting [[https://github.com/Electron100/butane/issues/21]]
 	 + incremental save
 	 + Many backreferences
-	 + Column renames in migrations
+	 + Column renames in migrations [[https://github.com/Electron100/butane/issues/89]]
 	 + prepared/reusable queries
```

Comment:
IMO this isnt a limitation - there are some performance gains here, but ... I doubt typical users of `butane` would see this as a priority.